package/MDAnalysis/topology/TOPParser.py

```diff
@@ -69,8 +70,12 @@
    by 18.2223.
 
    Chamber-style Amber topologies (i.e. topologies generated via parmed
-   conversion of a CHARMM topology to an AMBER one) are not currently supported.
-   Support will likely be added in future MDAnalysis releases.
+   conversion of a CHARMM topology to an AMBER one) are not currently
+   supported. Support will likely be added in future MDAnalysis releases.
+
+   As of version X.0.0, elements are no longer guessed if ATOMIC_NUMBER records
+   are missing. In those scenarios, if elements are necessary, users will have
+   to invoke the element guessers after parsing the topology file.
```

Comment:
Could you give an example or link to the relevant method?

Reply:
I've added a link to MDAnalysis.topology.guessers, and then added detailed example uses of element guessing there. If this is reasonable, I'll try to add a more detailed tutorial to the user guide in the long term?